README.md

```diff
@@ -57,6 +57,13 @@ public void onCreate() {
 ```
 
 #### 3.2 iOS project
+
+#### With Cocoapods
```

Comment:
👍

Reply:
I think `react-native link` will automatically add this to Podfile

https://facebook.github.io/react-native/docs/linking-libraries-ios#step-2